js_modules/dagit/packages/core/src/assets/usePartitionHealthData.tsx

```diff
@@ -39,7 +39,7 @@ export type PartitionHealthDimensionRange = {
   selected: string[];
 };
 
-async function loadPartitionHealthData(client: ApolloClient<any>, loadKey: AssetKey) {
+export async function loadPartitionHealthData(client: ApolloClient<any>, loadKey: AssetKey) {
```

Comment:
Since you're testing operations on the retrieved data, it might be simpler to export a function that encapsulates everything *after* the `client.query` call. Then you don't need to mock the `client.query` at all, you can just pass in your mocked objects.

Reply:
Ahh yeah let me switch to that, would be slightly cleaner than mocking the query function